ui/pages/Public/LandingPage.jsx

```diff
@@ -27,46 +37,68 @@ export default () =>
         <Grid.Column width={3} />
       </Grid>
     </Segment>
-    <Segment textAlign="center" size="big" secondary padded="very">
-      <Header size="large" content={<span>About <i>seqr</i></span>} />
-      <VerticalSpacer height={20} />
+    <PageSegment textAlign="center" size="big" secondary>
+      <Header size="medium">About <i>seqr</i></Header>
+      <VerticalSpacer height={10} />
       Next Generation Sequencing (NGS) is a powerful diagnostic and research tool for Mendelian disease, but without
-      proper tools, this data can be inaccessible to researchers. We are developing seqr as an open source web interface
+      proper tools, this data can be inaccessible to researchers. We have developed seqr as an open source web interface
       to make research productive, accessible, and user-friendly while leveraging resources and infrastructure at the
       Broad Institute.
-    </Segment>
-    <Segment size="large" padded="very">
+    </PageSegment>
+    <PageSegment size="large" padded="very">
       <Header
         textAlign="center"
         size="medium"
         content={
           <span>
-            Currently, <i>seqr</i> is in closed beta-testing mode and is only used by our group and collaborators
+            <i>seqr</i> is available through four methods:
           </span>}
       />
       <Grid>
         <Grid.Column width={3} />
         <Grid.Column width={10}>
-          <List bulleted>
+          <List ordered>
             <List.Item>
+              This instance is available for collaborators within the &nbsp;
+              <Anchor href="https://populationgenomics.org.au">Centre for Population Genomics</Anchor> with data pre-loaded into projects.
               If you are interested in collaborating with our group, please &nbsp;
               <a href="mailto:seqr@populationgenomics.org.au"><b>contact us</b></a>
```

Comment:
To me, these points (also below) look a bit odd without a period at the end. What do you think?

```suggestion
              <a href="mailto:seqr@populationgenomics.org.au"><b>contact us</b></a>.
```